_maps/shuttles/syndicate/syndicate_litieguai.dmm

```diff
@@ -2409,174 +2476,304 @@
 	},
-/obj/effect/turf_decal/siding/white/corner,
-/obj/effect/turf_decal/siding/white/corner{
+/obj/effect/turf_decal/spline/fancy/opaque/black{
 	dir = 4
 	},
-/obj/structure/cable{
-	icon_state = "2-4"
+/obj/effect/turf_decal/spline/fancy/opaque/black/corner{
+	dir = 8
 	},
-/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
-	dir = 6
+/turf/open/floor/plasteel/patterned/grid,
+/area/ship/storage)
+"UN" = (
+/obj/structure/cable,
+/obj/machinery/power/apc/auto_name/directional/west,
+/obj/machinery/light_switch{
+	dir = 4;
+	pixel_x = -20;
+	pixel_y = -11
 	},
-/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
-	dir = 5
+/turf/open/floor/carpet/cyan,
+/area/ship/bridge)
+"UO" = (
+/turf/closed/wall/mineral/titanium/nodiagonal,
+/area/ship/maintenance/starboard)
+"UX" = (
+/obj/machinery/atmospherics/components/unary/vent_scrubber/on/layer4{
+	dir = 8
+	},
+/obj/effect/turf_decal/siding/white{
+	dir = 1
 	},
-/turf/open/floor/vault,
+/obj/effect/turf_decal/siding/white,
+/obj/machinery/light/small/directional/east,
+/turf/open/floor/plasteel/tech/grid,
 /area/ship/storage)
-"Yk" = (
-/obj/structure/table,
-/obj/machinery/microwave{
-	pixel_y = 5
+"WB" = (
+/obj/machinery/atmospherics/components/unary/vent_pump/on/layer2,
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4,
+/obj/machinery/light/small/directional/west,
+/turf/open/floor/plasteel/freezer,
+/area/ship/crew/toilet)
+"WJ" = (
+/obj/structure/bed{
+	dir = 8
 	},
-/obj/machinery/light/small/directional/south,
-/turf/open/floor/plasteel/grimy,
-/area/ship/crew)
-"Yn" = (
-/obj/effect/spawner/structure/window/shuttle,
-/obj/structure/curtain/cloth{
-	color = "#ACD1E9"
+/obj/structure/curtain/bounty,
+/obj/item/bedsheet/syndie{
+	dir = 8
 	},
-/turf/open/floor/plating,
+/obj/machinery/airalarm/directional/north,
+/turf/open/floor/plasteel/dark,
 /area/ship/bridge)
-"Yv" = (
-/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
-	dir = 4
+"WP" = (
+/obj/effect/turf_decal/industrial/outline/red,
+/obj/structure/railing/corner{
+	dir = 8
 	},
-/obj/machinery/door/window{
-	dir = 4
+/obj/structure/rack,
+/obj/item/flashlight,
+/obj/item/flashlight,
+/obj/item/flashlight,
+/obj/machinery/airalarm/directional/east,
+/turf/open/floor/plasteel/tech/grid,
+/area/ship/storage)
+"WX" = (
+/obj/machinery/atmospherics/components/unary/vent_pump/on/layer2{
+	dir = 1
 	},
-/obj/machinery/door/firedoor/border_only{
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4,
+/obj/effect/turf_decal/spline/fancy/opaque/black{
 	dir = 4
 	},
 /obj/effect/turf_decal/spline/fancy/opaque/black{
-	dir = 1
+	dir = 8
 	},
-/turf/open/floor/plasteel/tech,
+/turf/open/floor/plasteel/patterned/brushed,
 /area/ship/medical)
-"YI" = (
-/obj/machinery/light/directional/west,
-/obj/structure/chair{
-	dir = 4
+"Xd" = (
+/obj/item/areaeditor/shuttle,
+/obj/item/reagent_containers/food/drinks/flask/gold,
+/obj/item/reagent_containers/food/drinks/bottle/whiskey,
```

Comment:
You should make this brandy or gin. This is a Cybersun captain, after all, no way they drink the same whiskey as the plebs. Too bad we don't have scotch.